Source/OxyPlot/OxyPlot.csproj

```diff
@@ -63,6 +63,7 @@
     <Compile Include="PlotModel\IPlotElement.cs" />
     <Compile Include="PlotModel\IPlotModel.cs" />
     <Compile Include="Rendering\OxySizeExtensions.cs" />
+    <Compile Include="Series\BarSeries\LinearBarSeries.cs" />
```

Comment:
the other project files in this directory should also be updated/kept in sync with this